### PR TITLE
Layer Panel tool-tip bug resolved

### PR DESCRIPTION
**Pull request by aPassie**
Layer panel name tooltip errors
Fixed Two errors :
1. The arrow pointing to the text loads in AFTER the rest of the tooltip
2 .Occasionally the tooltip will go too far to the right on hover. It's very inconsistent. Perhaps it's appending it to the end of the string location.

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
         "remove_tag": "./scripts/remove_tag.sh"
     },
     "dependencies": {
+        "@onlook/onlook": "file:",
```
Comment (Kitenite) [inline app/package.json:40]: There seems to be a lot of extra dependencies added to the PR. Can you remove these?

Comment (aPassie) [reply app/package.json:40]: ok

Comment (Kitenite) [reply app/package.json:40]: Yep, let me add some comments

File: backend/package.json [hunk 2]
```diff
@@ -15,5 +15,8 @@
   },
   "peerDependencies": {
     "typescript": "^5.0.0"
+  },
+  "dependencies": {
```
Comment (Kitenite) [inline backend/package.json:19]: This should not be added

File: cli/package.json [hunk 3]
```diff
@@ -41,6 +41,7 @@
     "dependencies": {
         "@onlook/utils": "^0.0.3",
         "commander": "^12.1.0",
+        "onlook": "file:",
```
Comment (Kitenite) [inline cli/package.json:44]: This should not be added

File: demos/babel/package.json [hunk 4]
```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "dependencies": {
+    "@onlook/babel-demo": "file:",
```
Comment (Kitenite) [inline demos/babel/package.json:6]: This should not be added

File: demos/next/package.json [hunk 5]
```diff
@@ -14,6 +14,7 @@
     "format:check": "prettier --check \"**/*.{ts,tsx,mdx}\" --cache"
   },
   "dependencies": {
+    "@onlook/next-demo": "file:",
```
Comment (Kitenite) [inline demos/next/package.json:17]: This should not be added

File: demos/remix/package.json [hunk 6]
```diff
@@ -20,7 +20,8 @@
     "isbot": "^4.1.0",
     "morgan": "^1.10.0",
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "remix": "file:"
```
Comment (Kitenite) [inline demos/remix/package.json:24]: This should not be added

File: demos/webpack/package.json [hunk 7]
```diff
@@ -13,7 +13,8 @@
   "license": "Apache-2.0",
   "dependencies": {
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-webpack-from-scratch": "file:"
```
Comment (Kitenite) [inline demos/webpack/package.json:17]: This should not be added

File: package.json [hunk 8]
```diff
@@ -21,5 +21,8 @@
     },
     "devDependencies": {
         "husky": "^9.0.11"
+    },
+    "dependencies": {
```
Comment (Kitenite) [inline package.json:25]: This should not be added

File: plugins/babel/package.json [hunk 9]
```diff
@@ -13,6 +13,7 @@
         "@babel/preset-react": "^7.23.3",
         "@babel/preset-typescript": "^7.24.1",
         "@babel/types": "^7.24.7",
+        "@onlook/babel-plugin-react": "file:",
```
Comment (Kitenite) [inline plugins/babel/package.json:16]: This should not be added

File: plugins/next/package.json [hunk 10]
```diff
@@ -22,5 +22,8 @@
     "author": "onlook",
     "devDependencies": {
         "nodemon": "^3.1.4"
+    },
+    "dependencies": {
```
Comment (Kitenite) [inline plugins/next/package.json:26]: This should not be added

File: plugins/next/package.json [hunk 11]
```diff
@@ -22,5 +22,8 @@
...
+        "@onlook/nextjs": "file:"
```
Comment (Kitenite) [inline plugins/next/package.json:27]: This should not be added

File: utils/package.json [hunk 12]
```diff
@@ -42,6 +42,7 @@
         "@babel/parser": "^7.14.3",
         "@babel/traverse": "^7.14.5",
         "@babel/types": "^7.14.5",
+        "@onlook/utils": "file:",
```
Comment (Kitenite) [inline utils/package.json:45]: This should not be added

**Review by Kitenite — CHANGES_REQUESTED** [review]
There seems to be some kind of automation that's adding dependencies that should be removed :)

**Comment by aPassie** [comment]
I am pulling a new request regarding the same.
Sorry for the inconvenience

**Comment by aPassie** [comment]
this is the new pull request : https://github.com/onlook-dev/onlook/pull/608

**Pull request closed without merging**